tests/testthat/test-server.R

```diff
@@ -623,6 +650,7 @@ test_that("404 pages have sensible schema", {
 })
 
 test_that("Error handler is triggered", {
+  testthat::skip("This is fixed now, but this test should be rewritten")
```

Comment:
This test no longer works because it relied on buggy behaviour in the zip serializer which is now fixed.

Reply:
Should we just remove it?